File: test/fixtures/output/java/okhttp/application-form-encoded.java
OkHttpClient client = new OkHttpClient();

MediaType mediaType = MediaType.parse("application/x-www-form-urlencoded");
RequestBody body = RequestBody.create(mediaType, "foo=bar&hello=world&hello%20world=world%20hello");
Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .post(body)
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .build();

Response response = client.newCall(request).execute();

File: test/fixtures/output/csharp/restsharp/application-form-encoded.cs
var client = new RestClient("http://mockbin.com/har");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar&hello=world&hello%20world=world%20hello", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
